test/unit/atomic_iteration_test.lua

```diff
@@ -36,7 +36,7 @@ function g.test_memtx()
     local function f(iterator)
         local transaction = {}
         -- old / new_tuple is not passed for vinyl
-        for request_number, old_tuple, new_tuple, space_id in iterator() do
+        for _, old_tuple, _, _ in iterator() do
```

Comment:
Why not just `for _, old_tuple in iterator() do`?

Reply:
When we replace var name with `_` we lost some information about items for the source code reader.
But I don't want to keep it as is and suppress luacheck warning inline. So I replaced variables to `_` but keep all four variables to notice for the reader that we have 4 variables on each loop.

Reply:
As for me, for saving information, in this case, the comment looks preferable, because now it seems like half of the developerfriendly behavior. "I will tell you the number of the elements, but I will not tell what those elements are".
This is just my opinion and I don't object if this piece of code stays "as is".
Approve was set earlier.

Reply:
Well, I keep only `_, old_tuple`.
Thanks for the review!